src/BaGet.Core/Search/DatabaseSearchService.cs

```diff
@@ -125,7 +125,12 @@ public async Task<DependentsResponse> FindDependentsAsync(string packageId, Canc
                 .OrderByDescending(p => p.Downloads)
                 .Where(p => p.Dependencies.Any(d => d.Id == packageId))
                 .Take(20)
-                .Select(p => p.Id)
+                .Select(r => new DependentResponse() {
```

Comment:
Please put `{` on a new line to follow the .NET Foundation's style:

```suggestion
                .Select(r => new DependentResponse
                {
```

(It's not very compact, I know 😅)